Frontend/ui-customer/src/Components/SettingSideBar/View.purs

```diff
@@ -94,17 +108,15 @@ logoutView state push =
   linearLayout
   [ height MATCH_PARENT
   , width MATCH_PARENT
-  ,  gravity BOTTOM
   , orientation VERTICAL
-  , padding (Padding 0 0 0 safeMarginBottom)
-  , margin (MarginBottom 10) 
+  , margin ( MarginBottom 100 )
   ][ linearLayout
       [ width MATCH_PARENT
       , height (V 1)
-      , background Color.greyDark
-      , margin (Margin 20 0 20 0)
+      , background Color.grey900
+      , margin ( MarginVertical 8 8 )
       ][]
-  , settingsMenuView {imageUrl : "ny_ic_logout,https://assets.juspay.in/nammayatri/images/user/ny_ic_logout.png", text : (getString LOGOUT_), tag : SETTINGS_LOGOUT} push
+  , settingsMenuView {imageUrl : "ic_logout,https://assets.juspay.in/nammayatri/images/user/ic_logout.png", text : (getString LOGOUT_), tag : SETTINGS_LOGOUT, iconUrl : ""} push
```

Comment:
why are we changing icons? are we updating the icons with some new ones?

Reply:
Yes, Aman asked to replace them with new icons.